open-metadata-resources/open-metadata-deployment/SampleData/README.md

```diff
@@ -1,7 +1,9 @@
+<!-- SPDX-License-Identifier: Apache-2.0 -->
```

Comment:
Copyright statement for documentation is 

<!-- SPDX-License-Identifier: CC-BY-4.0 -->

rather than Apache 2.0

Reply:
The copyright statement I am using is:
Copyright Contributors to the ODPi Egeria project.

in each file I changes (as best I can see!)
This is the same as being used in the top level README.md

What should it be if not this?

Reply:
<!-- SPDX-License-Identifier: CC-BY-4.0 -->
<!-- Copyright Contributors to the ODPi Egeria project. -->

However, you are right - there are many files still to be updated - I have a number of them done in my current workspace - just trying to at least get the new files set up correctly.

Reply:
To be accurate, the license is wrong not the copyright

Reply:
@mandy-chessell I cannot see any references to CC-BY-4.0 in the files here on github review.
I then used the 'command line tools' option to pull down direct from this pull request to a *new* branch (not the one I had worked on), and used 'grep against all the files under SampleData.
Again I could not find any reference. All SPDX license labels were using Apache-2.0 in SampleData, SampleData/db2 & SampleData/mariadb

I've included my log of this check below - can you see any errors?

It's possible you were very quick and reviewd before my latest changes - though they were all done yesterday and the review was today?

```
 ~/src/egeria/open-metadata-resources/open-metadata-deployment/charts/ranger [rangerdocker1|⚑ 24]
17:47 $ git checkout master
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/charts/ranger [master|⚑ 24]
17:47 $ git pull upstream master
remote: Enumerating objects: 80, done.
remote: Counting objects: 100% (58/58), done.
remote: Compressing objects: 100% (17/17), done.
remote: Total 34 (delta 12), reused 23 (delta 3), pack-reused 0
Unpacking objects: 100% (34/34), done.
From https://github.com/odpi/egeria
 * branch              master     -> FETCH_HEAD
   04aace21..7d527e2a  master     -> upstream/master
Updating 04aace21..7d527e2a
Fast-forward
 .../src/main/java/org/odpi/openmetadata/adapters/repositoryservices/inmemory/repositoryconnector/InMemoryOMRSMetadataCollection.java                         |  5 ++---
 .../src/main/java/org/odpi/openmetadata/repositoryservices/enterprise/repositoryconnector/EnterpriseOMRSMetadataCollection.java                              | 14 --------------
 .../igc-repository-proxy/src/main/java/org/odpi/openmetadata/virtualdataconnector/igc/connectors/eventmapper/IGCOMRSRepositoryEventMapper.java               | 18 ++++++++++--------
 .../virtual-data-connector/igc-repository-proxy/src/main/java/org/odpi/openmetadata/virtualdataconnector/igc/connectors/eventmapper/model/Constants.java     |  1 +
 .../igc-repository-proxy/src/main/java/org/odpi/openmetadata/virtualdataconnector/igc/connectors/repositoryconnector/IGCOMRSRepositoryConnector.java         |  6 ------
 5 files changed, 13 insertions(+), 31 deletions(-)
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/charts/ranger [master ↑·21|⚑ 24]
17:47 $ git checkout -b planetf1-SampleData1 master
Switched to a new branch 'planetf1-SampleData1'
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/charts/ranger [planetf1-SampleData1 L|⚑ 24]
17:48 $ git pull https://github.com/planetf1/egeria.git SampleData1
remote: Enumerating objects: 4, done.
remote: Counting objects: 100% (4/4), done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 2 (delta 1), reused 0 (delta 0), pack-reused 0
Unpacking objects: 100% (2/2), done.
From https://github.com/planetf1/egeria
 * branch              SampleData1 -> FETCH_HEAD
Updating 7d527e2a..ff0f3992
Fast-forward
 open-metadata-resources/open-metadata-deployment/SampleData/Coco Pharmaceuticals - Company Directory Database.xlsx  | Bin 0 -> 42217 bytes
 open-metadata-resources/open-metadata-deployment/SampleData/Coco Pharmaceuticals - EmpSalaryAnalysis Database.xlsx  | Bin 0 -> 35481 bytes
 open-metadata-resources/open-metadata-deployment/SampleData/Coco Pharmaceuticals - Employee Database.xlsx           | Bin 0 -> 34915 bytes
 open-metadata-resources/open-metadata-deployment/SampleData/Coco Pharmaceuticals - Location Database.xlsx           | Bin 0 -> 31097 bytes
 open-metadata-resources/open-metadata-deployment/SampleData/Coco Pharmaceuticals - Patient Database.xls             | Bin 0 -> 26112 bytes
 open-metadata-resources/open-metadata-deployment/SampleData/CompanyDirectoryDatabase-ContactEmail.csv               |  35 +++++++++++++++++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/CompanyDirectoryDatabase-ContactList.csv                |  27 +++++++++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/CompanyDirectoryDatabase-ContactPhone.csv               |  29 +++++++++++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.csv         |  20 ++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/EmployeeDatabase-Dept.csv                               |  10 ++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/EmployeeDatabase-Employee.csv                           |  20 ++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/LocationDatabase-WorkLocation.csv                       |   5 +++++
 open-metadata-resources/open-metadata-deployment/SampleData/PatientDatabase-PatientTable.csv                        |  33 +++++++++++++++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/README.md                                               |   9 +++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/db2/CompanyDirectoryDatabase-ContactEmail.sql           |  14 ++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/db2/CompanyDirectoryDatabase-ContactList.sql            |  20 ++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/db2/CompanyDirectoryDatabase-ContactPhone.sql           |  18 ++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/db2/EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.sql     |  39 +++++++++++++++++++++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/db2/EmployeeDatabase-Dept.sql                           |  15 +++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/db2/EmployeeDatabase-Employee.sql                       |  21 +++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/db2/LocationDatabase-WorkLocation.sql                   |  25 +++++++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/db2/PatientDatabase-PatientTable.sql                    |  21 +++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/db2/dropdb.sql1                                         |   7 +++++++
 open-metadata-resources/open-metadata-deployment/SampleData/db2/loadVDC.sh                                          |  13 +++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/gaiandb_config.properties.fragment                      |  31 +++++++++++++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/CompanyDirectoryDatabase-ContactEmail.sql       |  14 ++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/CompanyDirectoryDatabase-ContactList.sql        |  18 ++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/CompanyDirectoryDatabase-ContactPhone.sql       |  14 ++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.sql |  35 +++++++++++++++++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/EmployeeDatabase-Dept.sql                       |  14 ++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/EmployeeDatabase-Employee.sql                   |  19 +++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/LocationDatabase-WorkLocation.sql               |  23 +++++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/PatientDatabase-PatientTable.sql                |  20 ++++++++++++++++++++
 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/loadVDC.sh                                      |  20 ++++++++++++++++++++
 34 files changed, 589 insertions(+)
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/Coco Pharmaceuticals - Company Directory Database.xlsx
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/Coco Pharmaceuticals - EmpSalaryAnalysis Database.xlsx
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/Coco Pharmaceuticals - Employee Database.xlsx
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/Coco Pharmaceuticals - Location Database.xlsx
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/Coco Pharmaceuticals - Patient Database.xls
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/CompanyDirectoryDatabase-ContactEmail.csv
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/CompanyDirectoryDatabase-ContactList.csv
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/CompanyDirectoryDatabase-ContactPhone.csv
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.csv
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/EmployeeDatabase-Dept.csv
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/EmployeeDatabase-Employee.csv
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/LocationDatabase-WorkLocation.csv
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/PatientDatabase-PatientTable.csv
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/README.md
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/db2/CompanyDirectoryDatabase-ContactEmail.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/db2/CompanyDirectoryDatabase-ContactList.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/db2/CompanyDirectoryDatabase-ContactPhone.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/db2/EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/db2/EmployeeDatabase-Dept.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/db2/EmployeeDatabase-Employee.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/db2/LocationDatabase-WorkLocation.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/db2/PatientDatabase-PatientTable.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/db2/dropdb.sql1
 create mode 100755 open-metadata-resources/open-metadata-deployment/SampleData/db2/loadVDC.sh
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/gaiandb_config.properties.fragment
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/CompanyDirectoryDatabase-ContactEmail.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/CompanyDirectoryDatabase-ContactList.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/CompanyDirectoryDatabase-ContactPhone.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/EmployeeDatabase-Dept.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/EmployeeDatabase-Employee.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/LocationDatabase-WorkLocation.sql
 create mode 100644 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/PatientDatabase-PatientTable.sql
 create mode 100755 open-metadata-resources/open-metadata-deployment/SampleData/mariadb/loadVDC.sh
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/charts/ranger [planetf1-SampleData1 L|⚑ 24]
17:48 $ cd pen-metadata-resources/open-metadata-deployment/SampleData
-bash: cd: pen-metadata-resources/open-metadata-deployment/SampleData: No such file or directory
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/charts/ranger [planetf1-SampleData1 L|⚑ 24]
17:49 $ ls
Chart.yaml  README.md   chart.md    templates   values.yaml
^[[A✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/charts/ranger [planetf1-SampleData1 L|⚑ 24]
17:49 $ cd pen-metadata-resources/open-metadata-deployment/SampleData
^C
^[✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/charts/ranger [planetf1-SampleData1 L|⚑ 24]
17:49 $
set ✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/charts/ranger [planetf1-SampleData1 L|⚑ 24]
17:49 $ set -o vi
^[kk✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/charts/ranger [planetf1-SampleData1 L|⚑ 24]
17:49 $ ls
^[k✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/charts/ranger [planetf1-SampleData1 L|⚑ 24]
17:49 $ cd open-metadata-resources/open-metadata-deployment/SampleData
-bash: cd: open-metadata-resources/open-metadata-deployment/SampleData: No such file or directory
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/charts/ranger [planetf1-SampleData1 L|⚑ 24]
17:49 $ cd ../../..
✔ ~/src/egeria/open-metadata-resources [planetf1-SampleData1 L|⚑ 24]
17:49 $ cd ..
✔ ~/src/egeria [planetf1-SampleData1 L|⚑ 24]
17:49 $ cd open-metadata-resources/open-metadata-deployment/SampleData
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/SampleData [planetf1-SampleData1 L|⚑ 24]
17:49 $ ls
Coco Pharmaceuticals - Company Directory Database.xlsx CompanyDirectoryDatabase-ContactList.csv               PatientDatabase-PatientTable.csv
Coco Pharmaceuticals - EmpSalaryAnalysis Database.xlsx CompanyDirectoryDatabase-ContactPhone.csv              README.md
Coco Pharmaceuticals - Employee Database.xlsx          EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.csv        db2
Coco Pharmaceuticals - Location Database.xlsx          EmployeeDatabase-Dept.csv                              gaiandb_config.properties.fragment
Coco Pharmaceuticals - Patient Database.xls            EmployeeDatabase-Employee.csv                          mariadb
CompanyDirectoryDatabase-ContactEmail.csv              LocationDatabase-WorkLocation.csv
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/SampleData [planetf1-SampleData1 L|⚑ 24]
17:49 $ grep 4.0 *
Binary file Coco Pharmaceuticals - Patient Database.xls matches
CompanyDirectoryDatabase-ContactPhone.csv:24;M;44-0800-567-1234
EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.csv:0;3809890;9;7;Ivor;Padlock;499888;2373;8;Amsterdam;1;27;Chief Security Officer;1;86453;4000;1833;Wilhelmdreef;Amsterdam-Zuidoost;North Holland;1;1
EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.csv:0;3809890;9;6;Bob;Nitter;458109;3082;4;Amsterdam;1;15;Application Developer;0;34504;150;1833;Wilhelmdreef;Amsterdam-Zuidoost;North Holland;1;1
EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.csv:0;3809890;9;5;Faith;Broker;139870;2373;8;Amsterdam;1;27;Human Resources and Compliance Director;1;84355;4000;1833;Wilhelmdreef;Amsterdam-Zuidoost;North Holland;1;1
EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.csv:0;3809890;5;5;Reggie;Mint;188888;5656;8;London;2;27;Chief Finance Officer (CFO);1;125640;4100;32;Wibble Rd;Corterville;Greater London;1;2
EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.csv:0;3809890;2;7;Peter;Profile;986419;7432;5;London;2;27;Information Analyst;1;34520;450;32;Wibble Rd;Corterville;Greater London;1;2
EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.csv:0;3809890;0;5;Jules;Keeper;296776;5656;8;London;2;27;Chief Data Officer (CDO);1;11230;4000;32;Wibble Rd;Corterville;Greater London;1;2
grep: db2: Is a directory
grep: mariadb: Is a directory
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/SampleData [planetf1-SampleData1 L|⚑ 24]
17:49 $ cd db2
ls
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/SampleData/db2 [planetf1-SampleData1 L|⚑ 24]
17:49 $ ls
CompanyDirectoryDatabase-ContactEmail.sql       EmployeeDatabase-Dept.sql                       dropdb.sql1
CompanyDirectoryDatabase-ContactList.sql        EmployeeDatabase-Employee.sql                   loadVDC.sh
CompanyDirectoryDatabase-ContactPhone.sql       LocationDatabase-WorkLocation.sql
EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.sql PatientDatabase-PatientTable.sql
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/SampleData/db2 [planetf1-SampleData1 L|⚑ 24]
17:49 $ grep 4.0 *
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/SampleData/db2 [planetf1-SampleData1 L|⚑ 24]
17:49 $ grep SPDX *
CompanyDirectoryDatabase-ContactEmail.sql:-- SPDX-License-Identifier: Apache-2.0
CompanyDirectoryDatabase-ContactList.sql:-- SPDX-License-Identifier: Apache-2.0
CompanyDirectoryDatabase-ContactPhone.sql:-- SPDX-License-Identifier: Apache-2.0
EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.sql:-- SPDX-License-Identifier: Apache-2.0
EmployeeDatabase-Dept.sql:-- SPDX-License-Identifier: Apache-2.0
EmployeeDatabase-Employee.sql:-- SPDX-License-Identifier: Apache-2.0
LocationDatabase-WorkLocation.sql:-- SPDX-License-Identifier: Apache-2.0
PatientDatabase-PatientTable.sql:-- SPDX-License-Identifier: Apache-2.0
dropdb.sql1:-- SPDX-License-Identifier: Apache-2.0
loadVDC.sh:# SPDX-License-Identifier: Apache-2.0
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/SampleData/db2 [planetf1-SampleData1 L|⚑ 24]
17:49 $ cd ../mariadb/
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/SampleData/mariadb [planetf1-SampleData1 L|⚑ 24]
17:49 $ grep SPDX *
CompanyDirectoryDatabase-ContactEmail.sql:-- SPDX-License-Identifier: Apache-2.0
CompanyDirectoryDatabase-ContactList.sql:-- SPDX-License-Identifier: Apache-2.0
CompanyDirectoryDatabase-ContactPhone.sql:-- SPDX-License-Identifier: Apache-2.0
EmpSalaryAnalysisDatabase-EmpSalaryAnalysis.sql:-- SPDX-License-Identifier: Apache-2.0
EmployeeDatabase-Dept.sql:-- SPDX-License-Identifier: Apache-2.0
EmployeeDatabase-Employee.sql:-- SPDX-License-Identifier: Apache-2.0
LocationDatabase-WorkLocation.sql:-- SPDX-License-Identifier: Apache-2.0
PatientDatabase-PatientTable.sql:-- SPDX-License-Identifier: Apache-2.0
loadVDC.sh:# SPDX-License-Identifier: Apache-2.0
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/SampleData/mariadb [planetf1-SampleData1 L|⚑ 24]
17:49 $ cd ..
✔ ~/src/egeria/open-metadata-resources/open-metadata-deployment/SampleData [planetf1-SampleData1 L|⚑ 24]
17:50 $ grep SPDX *
README.md:<!-- SPDX-License-Identifier: Apache-2.0 -->
grep: db2: Is a directory
gaiandb_config.properties.fragment:# SPDX-License-Identifier: Apache-2.0
grep: mariadb: Is a directory
```

Reply:
The problem is that the README file is using the Apache 2.0 license and it should be using the CC-BY-4.0 license because it is documentation.